src/Microsoft.DocAsCode.Build.TableOfContents/TocItemInfo.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.DocAsCode.Build.TableOfContents
     using Microsoft.DocAsCode.DataContracts.Common;
     using Microsoft.DocAsCode.Plugins;
 
-    internal sealed class TocItemInfo
+    public sealed class TocItemInfo
```

Comment:
reason is TocResolverUtility.Resolve is using it and be public. Plan to change the return type to TocItemViewModel in next iteration

---
In reply to: [121313790](https://github.com/dotnet/docfx/pull/1750#discussion_r121313790) [](ancestors = 121313790)